notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>What kind of resource are you creating? Is this a dataset? Should we instead create a DATASET_NAME variable? Or is it a requirement of the tutorial that the dataset have the same name as the bucket?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Code was updated, unresolving because the sentence needs any update too :) </p><p> </p><p>Should now say something like</p><p>Set a name for your BigQuery Dataset and create it (or whatever you want for phrasing)</p>